websites/P/Puray Animes/presence.ts

```diff
@@ -0,0 +1,54 @@
+const presence = new Presence({
+		clientId: "972246349917610054"
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+presence.on("UpdateData", () => {
+	const { pathname } = window.location,
+		data: PresenceData = {
+			largeImageKey: "logo",
+			startTimestamp: browsingTimestamp
+		};
+	if (document.querySelector("input[id^=headlessui]"))
+		data.details = "Pesquisando Animes";
+	else if (pathname.startsWith("/home")) {
+		data.details = "Início";
+		data.state = "Navegando";
```

Comment:
Personally I don't think you need the two lines here and the first one should say what you're doing
```suggestion
		data.details = "Na página principal";
```